.changeset/config.json

```diff
@@ -1,6 +1,9 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
-  "changelog": "@changesets/cli/changelog",
+  "changelog": [
+    "@svitejs/changesets-changelog-github-compact",
+    { "repo": "vercel/vercel" }
+  ],
```

Comment:
Is this the correct syntax to have an array with first element a string and second element an object?

Reply:
Ya, it's similar to jest plugins. First is the package name, and next is the options. See [here](https://github.com/vercel/edge-runtime/blob/main/.changeset/config.json#L3-L6).